dmutils/forms/helpers.py

```diff
@@ -19,13 +19,61 @@ def remove_csrf_token(data):
 
 
 def get_errors_from_wtform(form):
-    """Converts errors from a Flask-WTForm into the same format we generate from content-loader forms. This allows us
-    to treat errors from both content-loader forms and wtforms the same way inside templates.
+    """Converts errors from a Flask-WTForm into the same format we generate from content-loader forms.
+
+    Returns a dictionary that includes three keys: `input_name`, `question`,
+    and `message`. The dictionary should be passed to the template as `errors`.
+
+    This allows us to treat errors from both content-loader forms and wtforms
+    the same way inside templates.
+
+    We also include in the dictionary errors in a format suitable for GOV.UK
+    Frontend components:
```

Comment:
I think we can rewrite this docstring to be less confusing. How about:

```
Converts Flask-WTForm errors into formats suitable for use in Digital Marketplace templates.

Returns a dictionary with the following keys, supporting three types of error display:

1) digitalmarketplace-frontend-toolkit template toolkit/forms/validation.html:
  `input_name`, `question`, `message`

2) govuk-frontend macro govukErrorSummary:
  `text`, `href`

3) govuk-frontend errorMessage:
  `errorMessage`

This allows us to treat errors from both content-loader forms and wtforms the same way inside templates, 
and provides backwards-compatible support for the migration to using govuk-frontend components.

Example usage:

        # app.py
        class Form(FlaskForm):
            input = DMTextInput()

        @flask.route("/")
        def view():
            form = Form()
            errors = get_errors_from_wtform()
            return render(
                "template.html",
                errors=errors,
            )

        # template.html  (govuk-frontend)
        {{ govukErrorSummary({
            "errorList": errors.values()
        }) }}

        {{ govukTextInput({
            "errorMessage": errors.input.errorMessage,
        }) }}

        # template.html (DM frontend toolkit uses 'errors' by default)
        {% include 'toolkit/forms/validation.html' %}        
```

Reply:
Nice idea 👍

Reply:
Thanks for writing that!